bukuserver/templates/bukuserver/lib.html

```diff
@@ -23,6 +23,14 @@
   <script>opener && (opener !== window) && document.body.classList.add('popup')</script>
 {% endmacro %}
 
+{% macro focus(location='body') %}
+  {% if location %}
+  <script>setTimeout(() => $('{{location|safe}} input:not([type=hidden])')[0]?.focus(), 500)</script>
+  {% else %}
+  <script>$(document).ready(() => document.activeElement?.blur())</script>
+  {% endif %}
+{% endmacro %}
```

Comment:
`focus(None)` cancels autofocus after the page is loaded; `focus()` waits for half a second and focuses on the first visible input, and `focus(selector)` does the same but limits the search to specified area.